openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +89,83 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+
+== Post installation steps
+
+To be able to SSH to any of the OCP cluster machines a _jump server_ is required.
+
+=== Install jump server
+
+[source,bash]
+----
+ansible-playbook ansible/playbook/openstack/openstack_vm_create_passwordstore.yml \
+  -e '{"openstack": {"vm": {"network": "ocp-sscpc-openshift","image": "Fedora-Cloud-Base-29", "flavor": "m1.small"}}}' \
+  -e vm_name=ocp-jump-server
+----
+
+Create a floating ip from the external network that your router is connected to, in our case `provider_net_cci_13`.
+
+[NOTE]
+====
+TBD: An ansible playbook that implements this.
+
+For now this needs to be done using the RHOS console.
+====
+
+[source,bash]
+----
+openstack floating ip create provider_net_cci_13
+----
+
+Associate the floating IP with the jump server.
+
+[NOTE]
+====
+TBD: An ansible playbook that implements this.
+
+For now this needs to be done using the RHOS console.
+====
+
+[source,bash]
+----
+openstack server add floating ip <uuid_of_the_create_server> <floating_ip_that_you_got_from_previous_command>
+----
+
+[WARNING]
+====
+Manually add an entry to the passwordstore under `openstack/ocp-jumpserver/floating_ip` with the floating IP. 
```

Comment:
As the floating IP will change (and this is perhaps the issue that we have today), we must take care about how to create such floating IP and renew it maybe ?